### PR TITLE
[HEXDEV-692] pojo model mismatch

### PR DESCRIPTION
**Pull request by wendycwong**
Two things added after Tomas code review:

1. Added dataset prostrate with null column name to make sure my fix is working.
2. Added Pyunit and Runit tests using the dataset.

### CODE REVIEW

File: h2o-r/tests/testdir_javapredict/runit_hexdev_692_pojo_model_prediction.R [hunk 1]
```diff
@@ -16,19 +16,22 @@ test <-
     # The objective of the test is to verify java code generation
     # for big models containing huge amount of trees.
     # This case verify multi-classifiers.
-    training_file <- test_file <- locate("smalldata/Training_Data.csv")
+    training_file <- test_file <- locate("smalldata/logreg/prostate_train_null_column_name.csv")
     #training_file <- test_file <- locate("smalldata/dd.csv")
-    
+    browser()
```
Comment (mmalohlava) [inline h2o-r/tests/testdir_javapredict/runit_hexdev_692_pojo_model_prediction.R]: are you sure about this one?

**Comment by mmalohlava** [comment]
@h2o-ops please test!

File: h2o-r/tests/testdir_javapredict/runit_hexdev_692_pojo_model_prediction.R (hunk 1 above)
Comment (wendycwong) [inline h2o-r/tests/testdir_javapredict/runit_hexdev_692_pojo_model_prediction.R]: Let me kill it.  Sorry.

**Comment by abal5** [comment]
@h2o-ops please test!

**Comment by abal5** [comment]
@h2o-ops please test!

**Comment by abal5** [comment]
@h2o-ops please test!

**Pull request merged**